sympy/solvers/solveset.py

```diff
@@ -1339,8 +1436,14 @@ def add_type(eq, symbol, domain):
         # check if it is exponential type equation
         elif _is_exponential(simplified_equation, symbol):
             new_eq = _solve_expo(simplified_equation, symbol)
-
-        if new_eq:
+        # check if it is logarithmic type equation
+        elif _is_logarithm_reducible_to_single_instance(
```

Comment:
Sometimes it's better not to follow pep8 strictly for better readability.

Reply:
No, I don't mean that. The name is fine (explicit is better than implicit). I am talking about the line breaks.

Reply:
Oh! :D cool I will change it

Reply:
Should I change the name of `is_exponential` too, since I have made some edits in it.

Reply:
Sure, whatever best describes it.